types/node/module.d.ts

```diff
@@ -280,13 +280,13 @@ declare module "module" {
              * The directory name of the current module. This is the same as the `path.dirname()` of the `import.meta.filename`.
              * **Caveat:** only present on `file:` modules.
              */
-            dirname?: string;
+            dirname: string;
```

Comment:
A few months back, I was happy to initially see this was defined correctly. Now checking back it's too bad this change was made because this is only available in `file:` modules and I'm not sure how to get TypeScript to make it possibly undefined after this is done globally.